Gemfile

```diff
@@ -1,17 +1,15 @@
-# A sample Gemfile
 source 'https://rubygems.org'
 
 group :lint do
+  gem 'foodcritic', '~> 3.0'
   gem 'foodcritic-rackspace-rules'
-  gem 'foodcritic'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.24'
 end
 
 group :unit do
-  gem 'berkshelf', '>= 3.0'
-  gem 'chefspec'
+  gem 'berkshelf', '~> 3'
+  gem 'chefspec', '~> 4'
   gem 'chef-sugar'
-  gem 'chef-zero'
 end
 
 group :kitchen_common do
```

Comment:
> `group :kitchen_common do`

To keep inline with `kitchen_rackspace` this is probably more accurately named:
`group :kitchen_vagrant do`